run/types/testing.ts

```diff
@@ -408,4 +409,9 @@ export type Id =
   | 'network.loki.messenger:id/openGroupTitleTextView'
   | 'Image picker'
   | 'network.loki.messenger:id/action_apply';
-  | 'Error message';
+
+export type testRisk = 
```

Comment:
this should be capitalized `TestRisk` as it is a type and not a variable name

Reply:
fixed this in https://github.com/oxen-io/session-appium/pull/18